pynamodb/attributes.py

```diff
@@ -72,6 +74,10 @@ def __init__(
     ) -> None:
         if default and default_for_new:
             raise ValueError("An attribute cannot have both default and default_for_new parameters")
+        if not callable(default) and not isinstance(default, _IMMUTABLE_TYPES):
+            raise ValueError("default must be immutable type or a callable")
```

Comment:
Thoughts on how we can help folks get to a fix quickly here? I was thinking we might want to suggest wrapping in a lambda or link to a docs page with some info

Reply:
Good point. Took the time to document `Attribute`'s initializer: 674f91e383ccd1e1a03dde31e96fd52fc31611f7

Reply:
nice, they look great